client/jetpack-cloud/style.scss

```diff
@@ -54,8 +54,8 @@
 	border-color: var( --studio-jetpack-green-50 );
 
 	&:hover {
-		background-color: var( --studio-jetpack-green-60 );
-		border-color: var( --studio-jetpack-green-60 );
+		background-color: var( --studio-jetpack-green-70 );
+		border-color: var( --studio-jetpack-green-70 );
```

Comment:
It also looks like Jetpack.com is using a transition effect on hover:

```
transition: background-color .1s;
```

Should we implement this on our links as well, so it even more 1:1 with the jetpack.com experience? 🙂

Reply:
Good call, @kallehauge ! Done in https://github.com/Automattic/wp-calypso/pull/61838/commits/0da9b955606195a9295a535cc7105c5e61e8a4cb